kqueen/kubeapi.py

```diff
@@ -29,31 +29,34 @@ def __init__(self, **kwargs):
             raise ValueError('Missing parameter cluster')
 
         logger.debug('Initialized KubernetesAPI for {}'.format(self.cluster))
-        self.kubeconfig_file = self.get_kubeconfig_file()
 
         # set apis
-        api_client = config.new_client_from_config(config_file=self.kubeconfig_file)
+        api_client = self.get_api_client()
+
         self.api_corev1 = client.CoreV1Api(api_client=api_client)
         self.api_extensionsv1beta1 = client.ExtensionsV1beta1Api(api_client=api_client)
         self.api_rbacauthorizationv1beta1 = client.RbacAuthorizationV1beta1Api(api_client=api_client)
         self.api_version = client.VersionApi(api_client=api_client)
 
-    def get_kubeconfig_file(self):
+    def get_api_client(self):
         """
-        Write kubeconfig file on filesystem.
+        Create Kubernetes API client with configuration from string.
 
         Returns:
             str: Kubeconfig file path
 
         """
-        # TODO: make configfile name random
-        configfile = '/tmp/kubernetes'
-        f = open(configfile, 'w')
+
+        # This super-ugly code configuration is cause by wrong desing of config-loading
```

Comment:
caused instead of cause, design instead of desing :man_student:

Reply:
It isn't a grammar but typo!

Reply:
Typo nazis then?